src/index.es.ts

```diff
@@ -10,18 +9,79 @@ import {
   getInsightsAnonymousUserToken,
 } from './helpers';
 import { createInfiniteHitsSessionStorageCache } from './lib/infiniteHitsCache';
+import { deprecate } from './lib/utils';
 
-const instantsearch = (options: InstantSearchOptions): InstantSearch =>
+interface InstantSearchModule {
+  (options: InstantSearchOptions): InstantSearch;
+  version: string;
+
+  // @major remove these in favour of the exports
+  /** @deprecated */
+  createInfiniteHitsSessionStorageCache: typeof createInfiniteHitsSessionStorageCache;
+  /** @deprecated */
+  highlight: typeof highlight;
+  /** @deprecated */
+  reverseHighlight: typeof reverseHighlight;
+  /** @deprecated */
+  snippet: typeof snippet;
+  /** @deprecated */
+  reverseSnippet: typeof reverseSnippet;
+  /** @deprecated */
+  insights: typeof insights;
+  /** @deprecated */
+  getInsightsAnonymousUserToken: typeof getInsightsAnonymousUserToken;
```

Comment:
We deprecate these two but don't export them.

Reply:
since they are already deprecated alltogether, I thought I shouldn't add them to  a new entry point, wdyt?

Reply:
Ah right, I forgot they were deprecated!